### PR TITLE
Added collaborator icon

### PR DESCRIPTION
**Pull request by abhijoshi2k**
## Type
What type of Pull request is this?

- [x] **New Icons**  
  Types: `SVG` 
- [ ] **Updating existing icons**  
  Icon: `_____` 
- [ ] **Small corrections**  
  Correcting typos, wrong licenses, file names, etc.

## Description

Added Icon for collaborator. It can be used for MemberEvent.

Closes #

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,13 +14,19 @@ This Repository aims to provide various SVG icons to use in your Readme, or any
...
+    - [Specific versions](#specific-versions)
+  - [Licenses](#licenses)
+  - [Why use this and not the actual source?](#why-use-this-and-not-the-actual-source)
+  - [Icons](#icons)
+    - [Octicons](#octicons)
+      - [Issues](#issues)
+      - [Pull requests](#pull-requests)
+      - [Miscelanious](#miscelanious)
+    - [Gifs](#gifs)
+  - [Submit Icons](#submit-icons)
```
Comment (Andre601) [inline README.md]: Please revert this back to the old ToC.

File: README.md [hunk 2]
```diff
@@ -124,6 +130,7 @@ All icons used are the 16px variant available, which fit perfectly in normal tex
 | [`UnwatchRepository.svg`][OcticonsUnwatchRepository] | ![OcticonsUnwatchRepository] |       |
 | [`WatchRepository.svg`][OcticonsWatchRepository]     | ![OcticonsWatchRepository]   |       |
 | [`Wiki.svg`][OcticonsWiki]                           | ![OcticonsWiki]              |       |
+| [`People.svg`][OcticonsPeople]                           | ![OcticonsPeople]              |       |
```
Comment (Andre601) [inline README.md]: Move this under `ForkedRepository.svg` to keep it alphabetical.
Also, remove 4 spaces from the second `|`

**Review by Andre601 — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -14,13 +14,19 @@ This Repository aims to provide various SVG icons to use in your Readme, or any
...
 ## Table of Contents
 
-- [How to use](#how-to-use)
-  - [Specific versions](#specific-versions)
-- [Licenses](#licenses)
-- [Why use this and not the actual source?](#why-use-this-and-not-the-actual-source)
-- [Icons](#icons)
-  - [Octicons](#octicons)
-- [Submit Icons](#submit-icons)
+- [Readme-Icons](#readme-icons)
```
Comment (PuneetGopinath) [inline README.md]: Andre will not accept `#readme-icons`, Don't remove until @Andre601 will say something.

**Review by PuneetGopinath — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (Andre601) [reply README.md]: Already gave my feedback.

File: README.md [hunk 4]
```diff
@@ -124,6 +124,7 @@ All icons used are the 16px variant available, which fit perfectly in normal tex
 | [`UnwatchRepository.svg`][OcticonsUnwatchRepository] | ![OcticonsUnwatchRepository] |       |
 | [`WatchRepository.svg`][OcticonsWatchRepository]     | ![OcticonsWatchRepository]   |       |
 | [`Wiki.svg`][OcticonsWiki]                           | ![OcticonsWiki]              |       |
+| [`People.svg`][OcticonsPeople]                       | ![OcticonsPeople]            |       |
```
Comment (Andre601) [inline README.md]: Also move it under `ForkedRepository.svg` please

**Review by Andre601 — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (PuneetGopinath) [reply README.md]: Hey @Andre601!
Head of docs, even I didn't understand the first time you said, he maybe not understood, you should have given detailed explanation the second time you said about it, something like: "cut the people row and add it under forked repo row"

Comment (abhijoshi2k) [reply README.md]: Done @PuneetGopinath

**Review by PuneetGopinath — APPROVED** [review]

**Review by Andre601 — APPROVED** [review]

**Pull request merged**